ducktape/tests/runner_client.py

```diff
@@ -157,7 +158,7 @@ def run(self):
     def setup_test(self):
         """start services etc"""
         self.log(logging.INFO, "Setting up...")
-        self.test.setUp()
+        self.test.setup()
```

Comment:
There does seem to be an assumption in the code that stderr is only used for minimal fatal error messages (I think I saw some place that only read stderr if the process exited abnormally), but in practice stderr might be used for debugging and thus have enormously huge output.